src/program.js

```diff
@@ -425,6 +428,11 @@ Example: $0 --help run.
         type: 'boolean',
         default: false,
       },
+      'runAsBinary': {
```

Comment:
I don't think it makes sense to define this as a command line option, do you? I can't think of any use for this.